bus_procurement_cost/dgs_data_cleaner.py

```diff
@@ -2,7 +2,7 @@
 import pandas as pd
 import shared_utils
 from calitp_data_analysis.sql import to_snakecase
-
+from bus_cost_utils import *
```

Comment:
I would change imports to be written either like `import bus_cost_utils` and use functions like `bus_cost_utils.new_prop_finder()` or only import specific functions  (if you really only need a couple things) like `from bus_cost_utils import new_prop_finder()`.

Generally, being clear on where a function is imported from is crucial, because as soon as you have multiple `utils` functions, you can easily lose track of where something is. In a universe with a lot of functions, writing `from pandas import * and from numpy import *` would create clashes because they might even share functions with the same name (sum, mean, etc)!`

If you can change your import statements, that would make functions / where they belong very clear!